### PR TITLE
[Feat] Github action to publish npm package

### PR DESCRIPTION
**Pull request by macrigiuseppe**
This PR will implement github action to publish Kepler.gl to npm and gpr once a new release is created

### CODE REVIEW

File: .github/workflows/npmpublish.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  publish-gpr:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://npm.pkg.github.com/
+          scope: '@macrigiuseppe'
```
Comment (macrigiuseppe) [inline .github/workflows/npmpublish.yml]: not sure this is necessary but it was part of the template

File: .github/workflows/npmpublish.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      - run: npm test
+
+  publish-npm:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
```
Comment (heshan0131) [inline .github/workflows/npmpublish.yml]: version 12?

Comment (macrigiuseppe) [reply .github/workflows/npmpublish.yml]: the template value provided by github action. I will update to our current version

Comment (macrigiuseppe) [reply .github/workflows/npmpublish.yml]: @heshan0131 updated

**Pull request merged**